TWLight/resources/models.py

```diff
@@ -277,7 +277,7 @@ class Meta:
         blank=True, null=True,
         # Translators: In the administrator interface, this text is help text for a field where staff can specify the standard duration of a manually granted account for this partner.
         help_text=_("The standard length of an access grant from this Partner. "
-            "Entered as <days hours:minutes:seconds>."
+            "Entered as " + "&lt" + "days hours:minutes:seconds" + "&gt."
```

Comment:
I wanted to isolate html entities from the others non html strings. But yes, both will work

Reply:
Ah, ok. Ideally it should all be one string, because this will get picked up for translation and translators need to see the entire sentence. Could you make that change? Then this should be good to merge :)

Reply:
sure. Done!